web/app.fluidity.money/app/queries/useVolumeTx.ts

```diff
@@ -23,113 +100,173 @@ export type VolumeTxsResponse = {
       }>;
     };
   };
+  errors?: unknown;
 };
 
 const useVolumeTxByAddressTimestamp = async (
+  network: string,
   // address: token symbol
   fluidAssets: { [address: string]: string },
   address: string,
-  iso8601Timestamp: string
+  iso8601Timestamp: string,
+  useMoralis = true
```

Comment:
Add this as an options object, that way if we have to use other services it'll be as easy as spreading the option into the options argument.